### PR TITLE
Barry PNB TrainTest

### PR DESCRIPTION
**Pull request by barry-ravichandran**
Hi Stephen,

This pull request contains the 15 VFold Train Script and the Test ipy notebook for the PNB data Artery Segmentation. 

Kindly review and provide your feedback.

Thank you,
Barry

### CODE REVIEW

**Comment by aylward** [comment]
Great code!

One training request...for random flipping during training, you should not flip on the y-axis.   Ultrasound images are direction dependent along the transducer lines.   The images are physically impossible if flipped on the y-axis.

Regarding the commit.  
1) Let's not create a copy of the itkARGUS code.  We should share that across tasks.  
2) The files in the linear_maps directory are specific to the PTX probe (Sonosite probe), so they shouldn't be duplicated in the PNB folder.  

If you could remove those from this commit - or make a new commit without them, that would be awesome.

**Comment by barry-ravichandran** [comment]
> Great code!
> 
> One training request...for random flipping during training, you should not flip on the y-axis. Ultrasound images are direction dependent along the transducer lines. The images are physically impossible if flipped on the y-axis.
> 
> Regarding the commit.
> 
> 1. Let's not create a copy of the itkARGUS code.  We should share that across tasks.
> 2. The files in the linear_maps directory are specific to the PTX probe (Sonosite probe), so they shouldn't be duplicated in the PNB folder.
> 
> If you could remove those from this commit - or make a new commit without them, that would be awesome.

The flipping I am doing in the training code in PNB/ARGUS/Train/ARUNet-3D-PR-Final15-VFold-training.py is across spatial_axis 0 and 2 (not 1). I did not do for spatial axis 1. 

I will remove the folders mentioned in (1) and (2) and make a new commit.

Thank you for your feedback!

**Comment by barry-ravichandran** [comment]
The latest commit removes all ARGUS related files. Only ARGUS/Train folder is retained.

**Comment by barry-ravichandran** [comment]
Made the ipynb in sync with the py training script and pushed the changes.

**Review by aylward — APPROVED** [review]
LGTM!!!
